### ISSUE TITLE
Doesn't function without 2.1 runtime that's reached end-of-life

### ISSUE THREAD

**Issue by atifaziz** [johtela/LiterateCS#8]
I installed using:

    dotnet tool install -g literatecs

which succeeded with the following output:

    You can invoke the tool using the following command: literatecs
    Tool 'literatecs' (version '1.0.40') was successfully installed.

However, attempting to execute the tool with `literatecs.exe -h` ended in the following error message:

    It was not possible to find any compatible framework version
    The framework 'Microsoft.NETCore.App', version '2.1.0' (x64) was not found.
    - The following frameworks were found:
        3.1.22 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
        5.0.13 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
        6.0.1 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]

    You can resolve the problem by installing the specified framework and/or SDK.

    The specified framework can be found at:
    - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=2.1.0&arch=x64&rid=win10-x64

The [2.1 runtime](https://dotnet.microsoft.com/en-us/download/dotnet/2.1) has reached [end-of-life](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core) in August of last year so I am hesitant to install it.

Is there any plan to update and publish the tool for the current LTS releases, 3.1 and 6.0?

**Comment by johtela** [comment]
LiterateCS has been without updates for long time, but nice to hear that somebody uses it.

I migrated the code to CLR 6.0 and uploaded it to Nuget. The version is now 2.0.43. Tested that it works with my repositories, but since so many dependencies was updated, there might be regression bugs I haven't found yet. If you encounter problems, let me know.

To update the tool run `dotnet tool update --global LiterateCS`
